ppr-api/migrations/versions/0bef490ea1d0_parties_split_columns.py

```diff
@@ -52,13 +50,20 @@ def downgrade():
     op.drop_index(op.f('ix_parties_first_name_split1'), table_name='parties')
 
     # Manually added
+    
     op.drop_index(op.f('party_last_name_gist_idx'), table_name='parties')
     op.drop_index(op.f('party_first_name_gist_idx'), table_name='parties')
-
+  
     op.drop_column('parties', 'first_name_split2')
     op.drop_column('parties', 'first_name_split1')
     op.drop_column('parties', 'last_name_split3')
     op.drop_column('parties', 'last_name_split2')
     op.drop_column('parties', 'last_name_split1')
 
+    op.drop_column('parties', 'first_name_char1')
+    op.drop_column('parties', 'first_name_char2')
+    op.drop_column('parties', 'first_name_key_char1')
+    op.drop_column('parties', 'bus_name_base')
+    op.drop_column('parties', 'bus_name_key_char1')
```

Comment:
hey steph, just verifying you meant to keep the changes in this migration?

It does look correct with this change

Reply:
Yeah I ran into this when I was trying to get the database going